src/LightningDB/LightningDB.csproj

```diff
@@ -4,13 +4,14 @@
     <Description>LightningDB</Description>
     <VersionPrefix>0.14.1</VersionPrefix>
     <Authors>Ilya Lukyanov;Corey Kaylor</Authors>
-    <TargetFrameworks>netstandard2.0;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net6.0;net7.0</TargetFrameworks>

```

Comment:
Libraries should target the smallest version it's compatible with, and the ones that have custom compilation constants usage. Libraries will use the TFM of the calling application. So if you reference a `netcoreapp3.1` compatible library and you run with `net7.0` then the runtime will be `net7.0`.
In this case I assume it's `netstandard2.0;netcoreapp3.1` because:
- there is an `#if` for netcoreapps
- netcoreapp3.1 is the smallest compatible version
- netstandard2.0 for cases out f the `#if`

Reply:
Also it's ok to keep the applications and test to target the TFMs you actually want to test/support, i.e. net6.0 and net7.0

Reply:
Awesome feedback! Thanks for taking the time to review. I'll submit another round based on your comments.